holoviews/plotting/util.py

```diff
@@ -515,8 +515,9 @@ def bokeh_palette_to_palette(cmap, ncolors=None):
         reverse = True
 
     # Some colormaps are inverted compared to matplotlib
-    inverted = not (categorical or cmap.capitalize() in palettes.mpl)
-    reverse = (reverse and not inverted) or inverted
+    inverted = (not categorical and not cmap.capitalize() in palettes.mpl)
+    if inverted:
+        reverse=not reverse
```

Comment:
I didn't mean to switch 518 around; that change has no effect.  The one on 519 is the important one.

Reply:
Are you definitely sure the new logic is correct? I tried exactly what you did and I was fairly sure that was inverting them in some cases. I'll have another look.

Reply:
I suppose you didn't touch the tests so those must be okay.

Reply:
Yeah, so I forgot to add a sequential colormap reverse test and that's what's failing for me now.

Reply:
Nevermind I hadn't pulled. I've added unit tests to cover each type of colormap now, just for consistency.